[2025-06-02T18:26:50.784+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_docker_dag.run_test_task scheduled__2023-03-08T00:00:00+00:00 [queued]>
[2025-06-02T18:26:50.799+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_docker_dag.run_test_task scheduled__2023-03-08T00:00:00+00:00 [queued]>
[2025-06-02T18:26:50.800+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-06-02T18:26:50.819+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_test_task> on 2023-03-08 00:00:00+00:00
[2025-06-02T18:26:50.829+0000] {standard_task_runner.py:60} INFO - Started process 375 to run task
[2025-06-02T18:26:50.835+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'test_docker_dag', 'run_test_task', 'scheduled__2023-03-08T00:00:00+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp7__1qd3g']
[2025-06-02T18:26:50.842+0000] {standard_task_runner.py:88} INFO - Job 82: Subtask run_test_task
[2025-06-02T18:26:50.955+0000] {task_command.py:423} INFO - Running <TaskInstance: test_docker_dag.run_test_task scheduled__2023-03-08T00:00:00+00:00 [running]> on host c22acf1bbdfe
[2025-06-02T18:26:51.146+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_docker_dag' AIRFLOW_CTX_TASK_ID='run_test_task' AIRFLOW_CTX_EXECUTION_DATE='2023-03-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-03-08T00:00:00+00:00'
[2025-06-02T18:26:51.151+0000] {logging_mixin.py:188} INFO - âœ… Airflow DAG ran successfully inside Docker! After updating
[2025-06-02T18:26:51.154+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-02T18:26:51.181+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=test_docker_dag, task_id=run_test_task, execution_date=20230308T000000, start_date=20250602T182650, end_date=20250602T182651
[2025-06-02T18:26:51.255+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-02T18:26:51.309+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
